_integration-test/ensure-backup-restore-works.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -ex
+
+source install/_lib.sh
+source install/dc-detect-version.sh
+
+echo "${_group}Test that backup/restore works..."
+echo "Creating backup..."
+$dcr -v $(pwd)/sentry:/sentry-data/backup --rm -T -e SENTRY_LOG_LEVEL=CRITICAL web export /sentry-data/backup/backup.json
```

Comment:
Probably need an ownership/permission set here. I suggest something like this:
```bash
mkdir $(pwd)/sentry/backup
chmod +w $(pwd)/sentry/backup
$dcr -v $(pwd)/sentry/backup:/sentry-data/backup --rm -T -e SENTRY_LOG_LEVEL=CRITICAL web export /sentry-data/backup/backup.json

Reply:
Darn looks like that isn't working, will play around and see if I can figure something out. Script seems to run fine locally though

Reply:
For some reason, write access for group + owner wasn't enough. I needed write access for all users

Reply:
Probably because the user and group IDs are different when running the sentry container (from the one GitHub actions bash script runner)

Reply:
Probably because the user and group IDs are different when running the sentry container (from the one GitHub actions bash script runner)

Reply:
ah that makes sense